src/frontend/src/pages/PIAList/messages.ts

```diff
@@ -0,0 +1,20 @@
+export default {
+  PiaListTitle: {
+    en: `List of PIAs`,
+  },
+  NoPias: {
+    H2Text: {
+      en: `You have no PIAs yet`,
+    },
+    FirstParagraph: {
+      en: `The PIA Intake questions ask for some basic information about the initiative you’re assessing in the PIA. If you get stuck on any of the questions, `,
+      LinkText: {
+        en: `ask your MPO for help.`,
+      },
+      LinkHref: `https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/privacy/resources/privacy-officers`,
```

Comment:
Wondering if markdowns - the way we used in `PPQFormPage/messages.ts` were better or this one.

what if the link text placement is not at the end in the other languages we support. What do you think?

Reply:
We will probably be getting rid of the markdown library, right? It is also kind of weird to use a markdown component for text bodies like that.

Reply:
Yeah, but until then shouldn't we be consistent? so that when we switch, it can be a drop-in replacement?

I am just thinking either we do it right for the future (that may require a discussion), or be consistent with what we have been doing. This seems like an another approach which will need revisit in addition to the MdEditor changes.